agent/command/s3_get.go

```diff
@@ -151,7 +127,7 @@ func (c *s3get) Execute(ctx context.Context,
 
 	// expand necessary params
 	if err := c.expandParams(conf); err != nil {
-		return errors.Wrap(err, "expanding params")
+		return errors.Wrap(err, "applying expansions")
```

Comment:
I know we are reverting but do we want to revert this error message as well? I think it makes sense not to in terms of single responsibility PR's but also it makes sense to to have better logging. I'm not sure if this revert is going to be 'unreverted' eventually / if the ticket is still going to be implemented though.

Reply:
LGTM but I am 2nd